components/DatePicker/datePicker.module.scss

```
@@ -22,16 +22,22 @@
   padding: 1rem;
   flex-direction: column;
   align-items: flex-start;
-  border-radius: .5rem;
-  background: var(--neutral-white, #FFF);
+  border-radius: 0.5rem;
+  background: $white;
+  width: 100%;
+  height: 100%;
+  border: 1px solid $gray300;
 
   /* Elevation/Normal */
-  box-shadow: 0rem .0625rem .125rem 0rem rgba(0, 0, 0, 0.12), 0rem 0rem .0625rem 0rem rgba(0, 0, 0, 0.08), 0rem 0rem .0625rem 0rem rgba(0, 0, 0, 0.08);
+  box-shadow:
+    0rem 0.0625rem 0.125rem 0rem rgba(0, 0, 0, 0.12),
+    0rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.08),
+    0rem 0rem 0.0625rem 0rem rgba(0, 0, 0, 0.08);
 
   .header {
     display: flex;
     width: 100%;
-    padding: 0rem .5rem .5rem .5rem;
+    padding: 0rem 0.5rem 0.5rem 0.5rem;
```

P4: 이렇게만 작성해도 될 것 같습니당

```suggestion
    padding: 0rem 0.5rem 0.5rem;
```

---

재영님이 만드신 컴포넌트인데 아직 달력쪽 디자인 시안이 다 나오지않아서 임시로 하신 것 같네요!

---

어라라... 재영님 파트였군여... 죄송함당...😢